packages/core/src/util/normalizeConfig.js

```diff
@@ -114,9 +112,7 @@ const defaults = {
     disableW3cTraceCorrelation: false,
     kafka: {
       traceCorrelation: true,
-      // Before we start phase 3 of the migration, 'binary' will be the default value. With phase 1, we will move to
-      // 'both',  with phase 2 it will no longer be configurable and will always use 'string'.
-      headerFormat: 'binary'
+      headerFormat: constants.kafkaHeaderFormatDefault
```

Comment:
Using the default value from `tracing/constants.js`, so we do not need to define it twice. This also made it necessary to move the type definition to `tracing/constants.js`.